lisa/executable.py

```diff
@@ -20,6 +21,11 @@
 T = TypeVar("T")
 
 
+# circular dep if we use the helper in node.py, unfortunately
+def _is_node_remote(node: Node) -> bool:
+    return isinstance(node.type_schema, RemoteNode)
```

Comment:
How about detect Node itself, not its schema? It's straightforward.

Reply:
Fair

Reply:
Not in this context, because it can't include node. The whole situation of Node referring Tool referring node again is not pretty and could get a fix someday.

Reply:
There may be another way. It uses some information of system, and uses shell a lot.

Reply:
This is for a future refactor, not now :)